src/styles/StyleUtils.js

```diff
@@ -733,6 +733,54 @@ function getReportWelcomeContainerStyle(isSmallScreenWidth) {
     };
 }
 
+/**
+ * Gets the correct styles for emoji suggestion item
```

Comment:
Not sure what we mean by `correct styles`?

Reply:
this is the style calculation for the Emoji Suggestion row
I updated the comment